book/src/user_manual/add_members.md

```diff
@@ -19,3 +19,11 @@ Members can also be added as a proposal (without the corresponding Commit messag
 ```
 
 In this case the the function returns an `MlsMessageOut` that needs to be fanned out to existing group members.
+
+# Retrieving a list of group members
```

Comment:
I think this should get its own file because it has nothing to do with adding members. We will probably also expose more functions like looking up members by KPR or so.